tests/ootb/install-1st-snap-and-core/task.yaml

```diff
@@ -0,0 +1,10 @@
+summary: Install first snap and core in one go
+# This test is disabled on Debian for now because it uncovers a bug.
+# https://forum.snapcraft.io/t/snapd-out-of-the-box-experience-debian-9
+systems: [-debian-*]
```

Comment:
Once 2.27 hits stable this can be removed. Lets remove it now and merge the PR once 2.27 is available in stable and the PR turns green because of this. WDYT?